### ISSUE TITLE
wrong icons and mis aligned eww 

### ISSUE THREAD

**Issue by al3rez** [siduck/chadwm#178]
![image](https://github.com/siduck/chadwm/assets/15009059/288ab352-3968-4519-a765-4dd1ef5ed497)

i'm using Pop Os! and a high-resolution laptop 2560x1440+0+0 and I get this look when  eww is open. How can I fix it?

**Comment by siduck** [comment]
nerdfont installed? the one used in eww config or change it to your nerdfont

**Comment by al3rez** [comment]
yeah the nerd font is installed, but i think it's not related. i'm using "SpaceMono Nerd Font"

**Comment by al3rez** [comment]
i think because your resolution is bigger, how can i fix it?

**Comment by siduck** [comment]
it is related to nerdfont, my resolution is 2k btw

**Comment by siduck** [comment]
![image](https://github.com/siduck/chadwm/assets/59060246/d65cbb5d-9990-4fdb-b0ce-d7b112de835d)

either replace all   instances of jetbrainsmono or install jetbrainsmono

**Comment by al3rez** [comment]
i managed it to get it look liek this

now 

![image](https://github.com/siduck/chadwm/assets/15009059/fd209da7-3886-4330-901f-3a98738ba66b)

**Comment by al3rez** [comment]
![image](https://github.com/siduck/chadwm/assets/15009059/4e678ed3-9322-4b85-b557-460b52be3f1a)
now like this, seems that the box at the bottom is cutting circles

**Comment by siduck** [comment]
![image](https://github.com/siduck/chadwm/assets/59060246/4f543ec0-0cd0-461e-99fe-999ad282b36f)

also the volume bars are based on a command that takes value from alsa, you can modify it for pulseaudio or pipewire, and the below line is for brightness, it uses xbacklight by default

**Comment by VehementHam** [comment]
I am having this issue as well

**Comment by siduck** [comment]
been ages since i used chadwm and this widget back, i'll comeback at some time

**Comment by VehementHam** [comment]
I've spent an hour trying to fix this, and haven't gotten anywhere. It could be a bug. All I know is that the bottom of the `radialBars` box is cut off even if I remove the footer. When switch the positions of the elements in the footer, the buttons cover the `radialBars` where they are being cut off.

When I switch the positions of the `radialBars` and the `time`, neither element is cut off, but both are too high.

The focal section is not as tall as it should be, and that is causing the bottom to be cut off or the whole thing to be to high. I've tried messing with some of the properties of the widgets, but I have had no luck.

Anyways, I wont worry about eww for now.
